contrib/IECoreUSD/src/IECoreUSD/USDScene.cpp

```diff
@@ -637,8 +637,16 @@ struct PrimVarConverter
 
 		if( !p )
 		{
+			/* todo check why this conversion fails
```

Comment:
Is this related to your recent conversation on usd-interest? Sounds like we need to check the return value of `m_primVar.Get()`? If so, let's get this fixed properly now.